### PR TITLE
Add PlistEdit Pro v1.8.2

### PR DESCRIPTION
**Pull request by NaanProphet**
Created new cask for PlistEdit Pro. Commercial software with 
14 day trial. Zap command to remove preferences. Tested on 
OS X 10.11.1 El Capitan.

### CODE REVIEW

File: Casks/plistedit-pro.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'plistedit-pro' do
+  version '1.8.2'
+  sha256 'd114975f724726189afbd8ac9f7113ff8c6de303ba7b892ce633b50f0951be49'
+
+  url 'http://www.fatcatsoftware.com/plisteditpro/PlistEditPro.zip'
+  name 'PlistEdit Pro'
+  homepage 'http://www.fatcatsoftware.com/plisteditpro/'
+  license :freemium
```
Comment (adityadalal924) [inline Casks/plistedit-pro.rb]: Should be `:commercial`

File: Casks/plistedit-pro.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  app 'PlistEdit Pro.app'
+
+  zap :delete => [
+    '~/Library/Preferences/com.fatcatsoftware.pledpro.plist',
+    '~/Library/Application Support/PlistEdit Pro'
+  ]
+
+  caveats 'After the 14 day trial period, files can still be opened but can no longer be saved.'
```
Comment (adityadalal924) [inline Casks/plistedit-pro.rb]: Remove this.

File: Casks/plistedit-pro.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+cask 'plistedit-pro' do
+  version '1.8.2'
+  sha256 'd114975f724726189afbd8ac9f7113ff8c6de303ba7b892ce633b50f0951be49'
```
Comment (adityadalal924) [inline Casks/plistedit-pro.rb]: version needs to be :latest, shasum :no_check, download URL is unversioned

File: Casks/plistedit-pro.rb (hunk 1 above)
Comment (jawshooah) [inline Casks/plistedit-pro.rb]: To clarify, `:freemium` is *not* applicable to software that has a limited trial period, then locks you out of any useful functionality. It is [specifically](https://github.com/caskroom/homebrew-cask/blob/cf8818dd74d169b4bdcf41893139c4ae15b099d3/doc/CASK_LANGUAGE_REFERENCE.md#valid-licenses) for apps that are "free-to-use, payment required for full or additional functionality".

**Comment by SirDap** [comment]
@adityadalal924 Thanks for the review comments! Slowly but surely learning

File: Casks/plistedit-pro.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  homepage 'http://www.fatcatsoftware.com/plisteditpro/'
+  license :commercial
+
+  app 'PlistEdit Pro.app'
+
+  zap :delete => [
+    '~/Library/Preferences/com.fatcatsoftware.pledpro.plist',
+    '~/Library/Application Support/PlistEdit Pro'
+  ]
+
```
Comment (jawshooah) [inline Casks/plistedit-pro.rb]: This empty line has to go as well.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8d377724a1232f3e13e6247849b1b8ad1d66e825. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**